itou/utils/tests.py

```diff
@@ -872,3 +882,77 @@ def test_valid_resume_link(self):
         form = ResumeFormMixin(data={"resume_link": resume_link})
         self.assertTrue(form.is_valid())
         self.assertFalse(form.has_error("resume_link"))
+
+
+class SupportRemarkAdminViewsTest(TestCase):
+    """
+    Test Support Remark admin views.
+    """
```

Comment:
commentaire inutile :)
les commentaires devraient éclairer un contexte (dire pourquoi, éclaircir une valeur obscure, etc) mais [pas dire ce que fait le code](https://blog.codinghorror.com/code-tells-you-how-comments-tell-you-why/)